fastlane/Fastfile

```diff
@@ -30,9 +30,23 @@ package = load_json(json_path: "./package.json")
   )
 end
 
+desc 'Automatically add commit message including build number.'
+private_lane :commit_version do |options|
+  package = load_json(json_path: "./package.json")
+  xcodeproj_path = 'ios/MindloggerMobile.xcodeproj'
+  version_number = package['version']
+  build_number = get_build_number(xcodeproj: xcodeproj_path)
+
+  git_commit(path: [
+    './android/app/build.gradle',
+    './ios/MindloggerMobile.xcodeproj/project.pbxproj'
+    ], message: "chore: bump version #{version_number} (#{build_number})"
+  )
+end
 
 desc 'Bump build numbers, and set the version to match the pacakage.json version.'
```

Comment:
Small typo
```suggestion
desc 'Bump build numbers, and set the version to match the package.json version.'
```

Reply:
Done! you can now pass `skipCommit:true` and `increment:x` like `yarn bump skipCommit:true increment:5`
Default behavior is to increment by 1 and generate the commit message